sim/midas/src/main/scala/midas/models/dram/Util.scala

```diff
@@ -265,6 +266,28 @@ trait HasAXI4Id extends HasNastiParameters { val id = UInt(nastiXIdBits.W) }
 trait HasAXI4IdAndLen extends HasAXI4Id { val len = UInt(nastiXLenBits.W) }
 trait HasReqMetaData extends HasAXI4IdAndLen { val addr = UInt(nastiXAddrBits.W) }
 
+class NastiAddressAttr (val addrBits: Int, val sizeBits: Int, val lenBits: Int)
+  (implicit val p: Parameters) extends Bundle { 
```

Comment:
Remove the `implicit val p` here, it is unneeded since there are no Nasti specific structures.

Reply:
Done